src/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -96,7 +79,9 @@ export class ArtworkApp extends React.Component<Props> {
       artwork: { listPrice, availability, is_offerable, is_acquireable },
     } = this.props
 
-    // Pageview
+    // FIXME: This breaks our global pageview tracking in the router level.
+    // Can these props be tracked on mount using our typical @track() or
+    // trackEvent() patterns as used in other apps?
```

Comment:
@mzikherman - How should we handle this? Maybe reach out to Ani?

Reply:
Yea, we should figure out if we need these in the page events, or if they can be inferred by the team later on.

The `trackProductView` call above here is also an ad-hoc tracking call that we want to fire on all navigations of an artwork page (such as from an artwork to another artwork), _if_ the resulting artwork is ecommerce.

In the tracking middleware, is it possible to reach into the props or Relay store or something, in order to figure out if we're navigating from an artwork to another artwork, and if so..include these additional properties/events?

Reply:
I guess we should remove this pageview tracking call (currently there's two that fire when navigating to an artwork, this one and the middleware one).